docs/contributing/featureflags.md

```diff
@@ -57,63 +66,127 @@ configuration too via `FlagDataFilePath`) with:
+output directory of the `Api` project. This is because the `/config` endpoint that clients use to
+query for feature flags is in `Api`. Doing this will ensure that the proper flag values get
+retrieved and sent to the client.
 
 :::
 
-Given the variety of clients and unique implementations, a few general rules:
+## Creating a new flag
 
-- Utilize the collection of feature states returned from the configuration API.
-- When in need of a feature state depend on what’s cached locally and available – this should
-  refresh frequently enough.
-- Maintain typed constants for feature flag keys.
+When beginning work on a new feature, discuss with your team whether it should be placed behind a
+feature flag. The team should agree on the scope of what is flagged and where the flag should be
+applied - both client-side and server-side. While there is no precise rule on what constitutes a
+"feature", work together on the best balance of flags and their respective purposes.
 
-## Creating a new flag
+### Local development
+
+As you begin work on the feature, use the `flags.json` data store to surface the flag to your
+consuming code to make sure that behavior is correct for all supported flag values. Since feature
+flags don’t have to exist in LaunchDarkly for initial development, **don’t create them online until
+you’re sure about the final implementation**.
 
-:::note
+When coding locally in the client codebase against a feature flag, your approach for retrieving
+feature flags differs based on the server instance you are querying.
 
-Compile-time configuration should be converted to use feature flags where appropriate. That said,
-feature flags are not to be used for permanent configuration – logic and storage should be developed
-to maintain this long-term.
+:::tip Local client development
+
+Keep in mind that for client local development, the source for the feature flag is dependent upon
+the server instance you're using. For example, if you are developing client-side code and
+referencing the QA Cloud Bitwarden API, the flag must be configured there and not in a local
+`flags.json` file.
 
 :::
 
-Talk with your team, manager / lead, and product about how and when to create new feature flags.
-Depending on how a project / solution is being delivered it may be best to use the online
-LaunchDarkly experience for setup. That said, LaunchDarkly is integrated into our Jira instance to
-make flag creation
-[easy](https://docs.launchdarkly.com/integrations/jira#creating-a-new-feature-flag-from-a-jira-issue).
+### Definition in LaunchDarkly
+
+In order to test the feature flag in any deployed environment, it must first be defined in the
+LaunchDarkly web app. To do this, request the flag from your Engineering Manager - they will have
+the appropriate access. You should discuss:
+
+- The data type of the flag
+- The default value of the flag
+- The possible values of the flag (for non-boolean types)
+- Any context-based rules that should drive flag behavior
```

Comment:
⛏️ I like to end bullets in parentheses (and there's another example below) but this is rather inconsistent across the docs.

Reply:
I think you meant period, right?  If so, I've added them.